### DISCUSSION TITLE
Release Candidate v1.0

### DISCUSSION THREAD

**Discussion by rabbibotton** [rabbibotton/clog#76] [Show and tell]
v1.0 I am reviewing and preparing the official 1.0 version. Less some documentation changes this is the version in QuickLisp the last few months.

<hr /><em>This discussion was created from the release <a href='https://github.com/rabbibotton/clog/releases/tag/v1.0-rc1'>Release Candidate v1.0</a>.</em>

**Comment by aykaramba** [comment]
Awesome, looking forward to it.
